mjidris/irondb
# Us 231

## Objective
Get React-to-Express requesting proxy working properly within Docker network.

## Detailed Description
In order to get proxying working for easier API calls in React, I had to set up the proxy to point to the API container on the internal Docker network. To see the primary code involved in this change, take a look at `client/package.json`. Towards the bottom, you'll see `http://api:3000`.

## Screenshot (if applicable)
N/A

## How to Test (if applicable)
After rebuilding the project, you'll be able to trace the network requests in the Chrome dev tools and see that the login request is being passed to `localhost:8001/login`, which is then proxied into the API. 

## Tests (if added)
N/A

## Requirements
[US-231](https://tree.taiga.io/project/cbrobsto-iron-db/us/231)
[T-252](https://tree.taiga.io/project/cbrobsto-iron-db/task/252)
